frontend/src/components/object/ObjectMetadataTagForm.vue

```diff
@@ -67,130 +72,134 @@ const onCancel = () => {
       :initial-values="initialValues"
       @submit="onSubmit"
     >
-      <div class="grid">
-        <div class="col-12">
-          <h2 class="font-bold">
-            Metadata
-          </h2>
-        </div>
-        <div class="grid col-11 pb-0 pt-0">
-          <div class="col-6">
-            Key
+      <span v-if="metadataEditable">
+        <div class="grid">
+          <div class="col-12">
+            <h2 class="font-bold">
+              Metadata
+            </h2>
           </div>
-          <div class="col-6">
-            Value
+          <div class="grid col-11 pb-0 pt-0">
+            <div class="col-6">
+              Key
+            </div>
+            <div class="col-6">
+              Value
+            </div>
           </div>
+          <div class="col" />
         </div>
-        <div class="col" />
-      </div>
-      <FieldArray
-        v-slot="{ fields, push, remove }"
-        name="metadata"
-      >
-        <div
-          v-for="(meta, index) of fields"
-          :key="'metadata.'+index"
-          class="grid"
+        <FieldArray
+          v-slot="{ fields, push, remove }"
+          name="metadata"
         >
-          <div class="grid col-11">
-            <div class="col">
-              <TextInput
-                :name="'metadata.'+index+'.key'"
-              />
+          <div
+            v-for="(meta, index) of fields"
+            :key="'metadata.'+index"
+            class="grid"
+          >
+            <div class="grid col-11">
+              <div class="col">
+                <TextInput
+                  :name="'metadata.'+index+'.key'"
```

Comment:
We should have spaces between the + for readability purposes. It's easy to miss this concatenation otherwise. Ideally we should try to favor using template strings instead of string concatenation, but unsure if that is the primary intent of assembly here.